ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/execution/versions/capella/WithdrawalSchema.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright ConsenSys Software Inc., 2022
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.spec.datastructures.execution.versions.capella;
+
+import tech.pegasys.teku.infrastructure.bytes.Bytes20;
+import tech.pegasys.teku.infrastructure.ssz.collections.SszByteVector;
+import tech.pegasys.teku.infrastructure.ssz.containers.ContainerSchema4;
+import tech.pegasys.teku.infrastructure.ssz.primitive.SszUInt256;
+import tech.pegasys.teku.infrastructure.ssz.primitive.SszUInt64;
+import tech.pegasys.teku.infrastructure.ssz.schema.SszPrimitiveSchemas;
+import tech.pegasys.teku.infrastructure.ssz.schema.collections.SszByteVectorSchema;
+import tech.pegasys.teku.infrastructure.ssz.tree.TreeNode;
+
+public class WithdrawalSchema
+    extends ContainerSchema4<Withdrawal, SszUInt256, SszUInt64, SszByteVector, SszUInt64> {
+
+  public WithdrawalSchema() {
+    super(
+        "Withdrawal",
+        namedSchema("index", SszPrimitiveSchemas.UINT256_SCHEMA),
+        namedSchema("validator_index", SszPrimitiveSchemas.UINT64_SCHEMA),
+        namedSchema("address", SszByteVectorSchema.create(Bytes20.SIZE)),
```

Comment:
we might be able to use `Eth1Address` here, I think that's ssz...

Reply:
actually no, bytes20 is probably right given the spec says bytes20 in Bellatrix... nevemind this one...